engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectModulesScreen.java

```diff
@@ -45,23 +45,13 @@
 import org.terasology.rendering.nui.databinding.Binding;
 import org.terasology.rendering.nui.databinding.ReadOnlyBinding;
 import org.terasology.rendering.nui.itemRendering.AbstractItemRenderer;
-import org.terasology.rendering.nui.widgets.ResettableUIText;
-import org.terasology.rendering.nui.widgets.TextChangeEventListener;
-import org.terasology.rendering.nui.widgets.UIButton;
-import org.terasology.rendering.nui.widgets.UILabel;
-import org.terasology.rendering.nui.widgets.UIList;
+import org.terasology.rendering.nui.widgets.*;
```

Comment:
Looks like your IntelliJ auto code cleanup is compressing independent imports to a wildcard import. Disable that
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports